demos/linear_mixed_system/linear_mixed_system.py.rst

```diff
@@ -0,0 +1,328 @@
+Linear mixed fluid-structure system
```

Comment:
This is a perfectly good title, however the name you are using for directories and files here is confusing. "linear_mixed_system" could refer to a huge range of finite element problems. Please use a name which indicates that this is fluid structure interaction.